microsoft/testsuites/core/vm_hot_resize.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.features.resize import Resize
+from lisa.schema import NodeSpace
+
+
+@TestSuiteMetadata(
+    area="vm_hot_resize",
+    category="functional",
+    description="""
+    This test suite tests vm behavior upon resizing without shutting down
+    """,
+)
+class VmHotResize(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case resizes the node and checks if it has the expected capabilities
+        (memory size and core count) after the resize
+
+        Steps:
+        1. Resize vm
+        2. Check the node's core count and memory size against their expected values
+        """,
+        priority=1,
+    )
+    def verify_vm_hot_resize(self, node: Node) -> None:
+        resize = node.features[Resize]
+        expected_vm_capability = resize.resize()
+        self._verify_cores_and_memory(node, expected_vm_capability)
+
+    def _verify_cores_and_memory(
+        self, node: Node, expected_vm_capability: NodeSpace
+    ) -> None:
+        expected_core_count = expected_vm_capability.core_count
+        actual_core_count = node.capability.core_count
```

Comment:
The actual core count should be checked by "lscpu" or other commands. We cannot check the LISA values.

Reply:
changed but it's causing errors

Reply:
Can you try on another machine? It may be some connectivity issue. If it's still repro, please file a bug with the log to me.

Reply:
I have tried Standard_DS12_v2, Standard_E8_v3, and Standard_A2_v2. Who should I file the bug under?

Reply:
Did you try to run LISA from another machine? It's not related to vm size, it should be related to the LISA environment.

Reply:
done